lib/converters.h

```diff
@@ -450,7 +450,7 @@ namespace _impl {
 
     inline void addTo(QUrlQuery& q, const QString& k, const QUrl& v)
     {
-        q.addQueryItem(k, v.toEncoded());
+        q.addQueryItem(k, QString::fromUtf8(v.toEncoded()));
```

Comment:
It's URL-encoded, so no Unicode.
```suggestion
        q.addQueryItem(k, QString::fromLatin1(v.toEncoded()));
```